xgcm/test/test_integrate.py

```diff
@@ -5,22 +5,26 @@
 from xgcm.test.datasets import datasets_grid_metric
 
 
-@pytest.mark.parametrize("gridtype", ["B", "C"])
-def test_integrate(gridtype):
+def _patch_dataset(grid_type):
+    """temporary fix for datasets_grid_metric"""
     ds_full = datasets_grid_metric()
-    ds = ds_full[gridtype]
-    grid = Grid(
-        ds,
-        coords=ds_full["coords"],
-        metrics=ds_full["metrics"],
-        # periodic=True,
-    )
+    ds = ds_full[grid_type]
+    return ds, ds_full["coords"], ds_full["metrics"]
+
```

Comment:
This is just me being lazy and not refactoring the metric dataset just yet. Mostly I want to speed this up in order to get #132 merged asap.